book.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :archived
+
+  @@books = []
+
+  def initialize(genre, author, label, publish_date, publisher, cover_state)
+    super(publish_date)
+    @genre = genre
+    @author = author
+    @label = label
+    @publisher = publisher
+    @cover_state = cover_state
+    @@books << self
+  end
+
+  def self.books
+    @@books
+  end
+
+  private
+
+  def can_be_archived?
+    return true if super == true or @cover_state == 'bad'
+  end
+end
+
+def list_all_books
+  Book.books.each do |book|
+    print "
+      publish_date: #{book.publish_date}
+      archived: #{book.archived}
+      publisher : #{book.publisher}
+      cover_state: #{book.cover_state}
+      "
+  end
+end
```

Comment:
Great work buddy. this is also going to close the display data issue. Nice work ✔ Closes #11